.github/workflows/check-if-migration-is-required.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   main:
-    name: is migration required
+    name: is migration/lockfile required
```

Comment:
similar to above, simpler is better - the `/` is also confusing

![image](https://user-images.githubusercontent.com/79684/115621073-df559500-a2aa-11eb-91aa-e7e9c4603210.png)